batch/scripts/append_data.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+# requires arguments
+# -d | --dataset
+# -v | --version
+# -s | --source
+# -D | --delimiter
+ME=$(basename "$0")
+. get_arguments.sh "$@"
+
+
+# Unescape TAB character
+if [ "$DELIMITER" == "\t" ]; then
+  DELIMITER=$(echo -e "\t")
+fi
+
+for uri in "${SRC[@]}"; do
+  aws s3 cp "${uri}" - | psql -c "COPY \"$DATASET\".\"$VERSION\" FROM STDIN WITH (FORMAT CSV, DELIMITER '$DELIMITER', HEADER)"
```

Comment:
You might need to list column names in target dataset explicitly here, since schemas no longer match
`COPY \"$DATASET\".\"$VERSION\" (column1, column2, column3, ...) FROM STDIN ...`

Best to put column names in double quote (same as dataset and version).

Reply:
Just seeing that you have this implemented in load_tabular_data below. Is this file still in use? If not delete